docs/how-to/deploy-kubernetes.md

```diff
@@ -0,0 +1,186 @@
+
+Adding the `configuration` options `registry_path` & `authentication_method` will add these additional steps to the workflow file.
+
+```json title="github-actions-plugin/settings | option default github token" showLineNumbers
+{
+    "registry" : "github",
+    "configuration" : {
+        "registry_path" : "amplication/example-kubernetes-deployment"
+    }
+}
+```
+
+```json title="github-actions-plugin/settings | option personal access token" showLineNumbers
+{
+    "registry" : "github",
+    "configuration" : {
+        "registry_path" : "amplication/example-kubernetes-deployment",
+        "authentication_method" : "pat"
+    }
+}
+```
+
+:::caution
+
+When specifying the `registry_path` keep in mind that it will still be suffixed by the service name. The path for package of for example the server component of a service named `logistics` will be `example-kubernetes-deployment/logistics` and live within the `amplication` GitHub organization.
+
+:::
+
+### Step 2: Configuration of the Helm Chart plugin
+
+After setting up the process that will result in the first artifact for the kubernetes deployment, we will continue with the setup for the Helm chart that will encapsulate the container image.
+
+When in the overview of the to be deployed service, navigate to the plugins part of the service configuration. Here the `Helm Chart` plugin can be found. Press the `Install` button, this should introduce pending changes and the option to press a `Settings` button, press this to be able to configure the plugin further. As with the `Github Actions` plugin some changes are made to the default generated json settings.
+
+The `root_level` setting determines whether the directory for the helm charts is placed at the root of the repository or in the base directory of the service. The `directory_name` setting determines what the sub-directory for the helm chart in the root level or service base directory is called.
+
+```json title="helm-chart-plugin/settings" showLineNumbers
+{
+  "root_level": true,
+  "directory_name": "helm",
+  "server": {
+    "chart_version": "0.0.1",
+    "application_version": "0.0.1",
+    "repository": "ghcr.io/amplication/example-kubernetes-deployment/logists",
+    "tag": "latest",
+    "host": "server.example.com",
+    "port": "3000"
+  },
+  "admin_ui": {
+    "enabled": false
+  }
+}
+```
+
+:::note
+
+Everything that is in the environments variable file for the service is moved to the `configmap` part of the helm chart, it would be advised to move secret related configuration to the secrets object and preferably not have the secrets stored in the generated code at all (as this is implementation specific the decision was made to add everything to the `configmap`).
```

Comment:
```suggestion
Everything that is in the environments variable file for the service is moved to the `configmap` part of the helm chart. As this might contain also secrets, we advise to move secret related configuration to the secrets object and preferably not have the secrets stored in the generated code at all (as this is implementation specific the decision was made to add everything to the `configmap`).
```